extensions/bitwarden/src/api.ts

```diff
@@ -1,23 +1,26 @@
 import { getPreferenceValues } from "@raycast/api";
 import execa from "execa";
 import { existsSync } from "fs";
-import which from "which";
+import { dirname } from "path/posix";
```

Comment:
any reason to use `path/posix` instead of just `path`? If we ever want to be multi platform (granted it won't work with home-brew anyway), `path` will work out of the box